### ISSUE TITLE
Move project related information to the team discussion page

### ISSUE THREAD

**Issue by clarencecastillo** [ntuoss/Sparrow-web#2]
I suggest that we keep READMEs scoped to repository particulars such as **setting up**, **deployment** or **troubleshooting**. We should move project related information such as general development resources, tools, tutorials or project specifications to @ntuoss/sparrows

**Comment by ZaynJarvis** [comment]
I leave only the tech specs in README cuz the real set up and deployment I will try to let them handle it.
Is it a strategy I can use for this beginner project?

**Comment by clarencecastillo** [comment]
Yeah that's just one way we can use the tools made available to us by segregating information according to their function. Repository specific info in repository, project info in project. Your call mate 👍
